build.sbt

```diff
@@ -14,26 +15,24 @@ lazy val scalaModels = project.in(file("."))
       "com.thesamet.scalapb" %% "scalapb-runtime" % scalapb.compiler.Version.scalapbVersion % "protobuf"
     ),
 
+    Compile / scalacOptions ++= Seq("-release:11"),
+
     Compile / PB.targets := Seq(
       scalapb.gen() -> (Compile / sourceManaged).value / "scalapb"
     ),
 
     Compile / PB.protoSources := Seq(baseDirectory.value / "./proto"),
     releaseCrossBuild := true,
-    releaseProcess := {
-      val process = Seq[ReleaseStep](
-        checkSnapshotDependencies,
-        inquireVersions,
-        runClean,
-        runTest,
-        setReleaseVersion,
-        releaseStepCommandAndRemaining("+publishSigned"),
-      )
-
-      if (!isSnapshot.value) {
-        process ++ Seq[ReleaseStep](releaseStepCommand("sonatypeBundleRelease"))
-      } else {
-        process
-      }
-    }
+    releaseVersion := fromAggregatedAssessedCompatibilityWithLatestRelease().value,
```

Comment:
For this specific library, where version numbers have [a very specific meaning agreed with the iOS and Android developers](https://github.com/guardian/mobile-apps-api/blob/main/doc/versioning-of-blueprint-endpoints.md), we want close manual control, rather than automated version-numbering that is focussed on _Scala_ compatibility (where version numbers might jump too quickly). 

Note that diamond-dependency problems probably won't happen, because the only Scala consumer of this library references it directly, rather than via another library (or libraries).